src/services/user.service.js

```diff
@@ -0,0 +1,36 @@
+const { User } = require('../models/User.model');
+const { generateRandomId } = require('../Random');
+
+const getAll = async () => {
+  return User.findAll();
+};
+
+const getById = async (id) => {
+  return User.findByPk(+id);
+};
+
+const updateUser = async ({ id, name }) => {
+  await User.update({ name }, { where: { id } });
+};
+
+const createUser = async (name) => {
+  const id = generateRandomId();
+
+  return User.create({ id, name });
+};
+
+const deleteUser = async (id) => {
+  await User.destroy({
+    where: {
+      id,
+    },
+  });
```

Comment:
Consider checking if the user with the given `id` actually exists before attempting to delete. This can help provide more informative feedback if the user does not exist.